hw5-1/main.c

```diff
@@ -0,0 +1,53 @@
+#include <stdio.h>
+#include "stack.h"
+
+int stackExpressionCount(const char *expression, int amountOfChars) {
```

Comment:
```suggestion
int calculateExpression(const char *expression, int amountOfChars) {
```
Мы ведь вычисляем выражение, а не количество считаем.
Также название должно отражать смысл функции, а не используемые структуры. Поэтому стек упоминать не нужно